lib/stdlib/test/lists_SUITE.erl

```diff
@@ -2333,6 +2334,21 @@ flatten_2(Config) when is_list(Config) ->
 flatten_2_e(Config) when is_list(Config) ->
     ok.
 
+%% Test lists:transpose/1.
+transpose(Config) when is_list(Config) ->
+    [] = lists:transpose([[], [], [], []]),
+    [[a0, b0, c0], [a1, b1, c1], [a2, b2, c2]] =
+        lists:transpose([[a0, a1, a2], [b0, b1, b2], [c0, c1, c2]]),
+    [[a0, b0], [a1, b1], [a2, b2]] =
+        lists:transpose([[a0, a1, a2], [b0, b1, b2]]),
+
+    %% Error cases.
+    {'EXIT',{function_clause,_}} = (catch lists:transpose([], [b])),
+    {'EXIT',{function_clause,_}} = (catch lists:transpose([a], [])),
+    {'EXIT',{function_clause,_}} = (catch lists:transpose([a], [b,c])),
```

Comment:
These test cases fail. Did you actually run the test suite?

Reply:
@bjorng  I don't know why the tests cases take ages on my laptop and they are failing in different parts :-/ I was checking with Actions here... and looks like all tests are passing, right?

Reply:
The Gihub actions only run a very small subset of test cases, essentially a smoke test.

Have you looked at https://github.com/erlang/otp/wiki/Running-tests and how you can run individual tests? So after the initial building, you can do:

```
1> ts:run(stdlib, lists_SUITE, [batch]).
```

or even:

```
1> ts:run(stdlib, lists_SUITE, transpose, [batch]).
```

Reply:
done :-)